config/processors/syslog_log_audit_checkpoint.operations.conf

```diff
@@ -38,7 +38,7 @@ filter {
     }
   }
 
-  if [rest_msg] =~ "illegal pam password attempt" {
+  if [rest_msg] =~ "illegal (pam|PAM) password attempt" {
```

Comment:
@Akhila-Y This is unrelated. Please revert this change in this PR. You can create another PR to add this change.